GitUI/UserControls/FilterToolBar.Designer.cs

```diff
@@ -264,8 +264,8 @@ private void InitializeComponent()
         private ToolStripMenuItem tsmiAuthorFilter;
         private ToolStripMenuItem tsmiDiffContainsFilter;
         private ToolStripMenuItem tsmiHash;
-        private ToolStripButton tsmiShowReflogs;
-        private ToolStripButton tsmiShowFirstParent;
+        private GitUI.CommandsDialogs.ToggleToolStripPushButton tsmiShowReflogs;
+        private GitUI.CommandsDialogs.ToggleToolStripPushButton tsmiShowFirstParent;
         private ToolStripTextBox tstxtRevisionFilter;
         private ToolStripLabel tslblRevisionFilter;
         private ToolStripSeparator toolStripSeparator19;
```

Comment:
`tsbtnAdvancedFilter` requires this too from #9728 (that adds push in #9728)

Reply:
@gerhardol : good catch, I focused on usages of `Checked` property, but this button is somewhat different. However, it is actually a toggle.
I'll have to work a little bit on determining if it has to be pressed or not, as I do not see a property return a consistent value about the presence of advanced filters if the branch filter is applied or not (the branch filter seems to be integrated in the filters, making the `HasFilter` property `true`, once advanced filter has been activated once with this setting enabled.

Reply:
As this is added in #9728 (plan to merge today), it is not visible yet. Need to be added before merge.